docs/core/compatibility/index.md

```diff
@@ -55,7 +55,13 @@ Changes in this category modify the public surface area of a type. Most of the c
 
 - ❌ **DISALLOWED: Renaming or removing a public type**
 
-   This breaks all code that uses the renamed or removed type.
+  This breaks all code that uses the renamed or removed type.
+
+  .NET's policy for obsoleting APIs is as follows:
+
+  > An API that shipped in a long-term support (LTS) release must be obsoleted in the subsequent LTS release before it can be removed. In rare cases, exceptions are made to obsolete an API before the subsequent LTS release based on business needs. These obsoletions will be documented and communicated to customers.
```

Comment:
```suggestion
  > An API that shipped in a long-term support (LTS) release must be obsoleted in the subsequent LTS release before it can be removed. In rare cases, exceptions are made to obsolete an API before the subsequent LTS release based on business needs. All obsoletions are documented and communicated to customers.
```